rotkehlchen/api/v1/schemas.py

```diff
@@ -2775,3 +2775,7 @@ def make_nft_filter_query(  # pylint: disable=no-self-use
             'ignore_cache': data['ignore_cache'],
             'filter_query': filter_query,
         }
+
+
+class MultipleAddressAsyncSchema(AsyncQueryArgumentSchema):
+    addresses = DelimitedOrNormalList(EthereumAddressField(), load_default=None)
```

Comment:
Also I guess this schema should become a parent for this one https://github.com/rotki/rotki/blob/eb51706e9e59f0cf5627e499b7c1e55032eb6059/rotkehlchen/api/v1/schemas.py#L2408

Reply:
I have abstracted it